tests/tests_pytorch/test_cli.py

```diff
@@ -556,9 +556,8 @@ def add_arguments_to_parser(self, parser):
     with mock.patch("sys.argv", ["any.py"] + cli_args):
         cli = CkptPathCLI(BoringCkptPathModel, subclass_mode_model=True)
 
-    assert isinstance(cli.model, BoringCkptPathSubclass)
+    assert not isinstance(cli.model, BoringCkptPathSubclass)
     assert cli.model.hidden_dim == 8
-    assert cli.model.extra is True
```

Comment:
Why did this change? Old behaviors should not change, unless there was bug. Was there a bug? If not, revert back.